src/gpuarray_buffer_cuda.c

```diff
@@ -1305,6 +1305,10 @@ static int cuda_callkernel(gpukernel *k, unsigned int n,
     }
 
     switch (n) {
+    case 0:
+      ctx->err = cuLaunchKernel(k->k, 1, 1, 1, 1, 1, 1, shared,
+                                ctx->s, args, NULL);
+      break;
```

Comment:
I'm not sure we want to support 0-dim at this level.  This is basically useless.

If you really need it special case it where you do the kernel call.